lib/index.ts

```diff
@@ -43,18 +48,21 @@ function buildDepTreeFromFiles(
   }
 
   const manifestFileFullPath = path.resolve(root, manifestFilePath);
+
   if (!fs.existsSync(manifestFileFullPath)) {
-    throw new Error(`Neither packages.config nor .csproj file found at location: ${manifestFileFullPath}`);
+    throw new Error('Neither packages.config nor ' +
+      `${PROJ_FILE_EXTENSION.join(', ')} files found at location: ${manifestFileFullPath}`);
   }
 
   const manifestFileContents = fs.readFileSync(manifestFileFullPath, 'utf-8');
+  const manifestFileExtension = path.extname(manifestFileFullPath);
 
-  if (_.endsWith(manifestFilePath, '.csproj')) {
+  if (_.includes(PROJ_FILE_EXTENSION, manifestFileExtension)) {
     return buildDepTreeFromCsproj(manifestFileContents, includeDev);
   } else if (_.endsWith(manifestFilePath, 'packages.config')) {
     return buildDepTreeFromPackagesConfig(manifestFileContents, includeDev);
   } else {
-    throw new Error(`Unsupported file ${manifestFilePath},
-    'Please provide either packages.config or .csproj file.`);
+    throw new Error(`Unsupported file ${manifestFilePath}, Please provide ` +
+      `either packages.config or one of ${PROJ_FILE_EXTENSION.join(', ')} files.`);
```

Comment:
I think we want to stay away from dynamic errors, they will be hard to catch can we maybe just say a project file / 'proj' file? and this can cover them all and keep the error the same